docs/user_guide/hopsworks/jupyter.rst

```diff
@@ -0,0 +1,227 @@
+    :target: `jupyter8.png`_
+    :align: center
+    :figclass: align-center
+
+    Creating a spark dataframe
+
+**Step 2 : Download the Spark Dataframe to a local Pandas Dataframe using %%sql or %%spark**:
+
+Note: you should **not** try to download large spark dataframes for plotting. When you plot a dataframe, the entire dataframe must fit into memory, so add the flag `--maxrows x` to limit the dataframe size when you download it to the local Jupyter server for plotting.
+
+Using %%sql:
+
+.. _jupyter9.png: ../../_images/jupyter9.png
+.. figure:: ../../imgs/jupyter9.png
+    :alt: Downloading the spark dataframe to a pandas dataframe using %%sql
+    :target: `jupyter9.png`_
+    :align: center
+    :figclass: align-center
+
+    Downloading the spark dataframe to a pandas dataframe using %%sql
+
+Using %%spark:
+
+.. _jupyter10.png: ../../_images/jupyter10.png
+.. figure:: ../../imgs/jupyter10.png
+    :alt: Downloading the spark dataframe to a pandas dataframe using %%spark
+    :target: `jupyter10.png`_
+    :align: center
+    :figclass: align-center
+
+    Downloading the spark dataframe to a pandas dataframe using %%spark
+
+
+**Step 3 : Plot the pandas dataframe using Python plotting libraries**:
+
+When you download a dataframe from spark to pandas with sparkmagic, it gives you a default visualization of the data using autovizwidget, as you saw in the screenshots above. However, sometimes you want custom plots, using matplotlib or seaborn. To do this, use the sparkmagic %%local to access the local pandas dataframe and then you can plot like usual. Just make sure that you have your plotting libraries (e.g matplotlib or seaborn) installed on the Jupyter machine, contact a system administrator if this is not already installed.
+
+.. _jupyter13.png: ../../_images/jupyter13.png
+.. figure:: ../../imgs/jupyter13.png
+    :alt: Import plotting libraries locally on the Jupyter server
+    :target: `jupyter13.png`_
+    :align: center
+    :figclass: align-center
+
+    Import plotting libraries locally on the Jupyter server
+
+.. _jupyter14.png: ../../_images/jupyter14.png
+.. figure:: ../../imgs/jupyter14.png
+    :alt: Plot a local pandas dataframe using seaborn and the magic %%local
+    :target: `jupyter14.png`_
+    :align: center
+    :figclass: align-center
+
+    Plot a local pandas dataframe using seaborn and the magic %%local
+
+.. _jupyter15.png: ../../_images/jupyter15.png
+.. figure:: ../../imgs/jupyter15.png
+    :alt: Plot a local pandas dataframe using matplotlib and the magic %%local
+    :target: `jupyter15.png`_
+    :align: center
+    :figclass: align-center
+
+    Plot a local pandas dataframe using matplotlib and the magic %%local
+
+
+Want to Learn More?
+-------------------
+
+We have provided a large number of example notebooks, available here_, upload them to your project on Hopsworks and try it out! You can also have a look at HopsML (hops-tensorflow_), which enables large-scale distributed deep learning on Hops.
```

Comment:
@Limmen maybe we can also mention that these notebooks are already available as part of the Deep Learning tour?

Reply:
Agree, fixed